fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerUniqueIndexTest.java

```diff
@@ -366,4 +371,184 @@ public void resolveUniquenessViolations() {
             context.commit();
         }
     }
+
+    @Test
+    public void testReadableUniquePendingSingle() {
```

Comment:
Similarly, it would be good to make sure that we test building a unique index from a non-unique index both with and without violations, and erroring or not.

Reply:
✅